packages/@n8n/benchmark/scripts/n8n-setups/scaling-single-main/docker-compose.yml

```diff
@@ -109,6 +111,7 @@ services:
       - DB_TYPE=postgresdb
       - DB_POSTGRESDB_HOST=postgres
       - DB_POSTGRESDB_PASSWORD=password
+      - N8N_RUNNERS_ENABLED=true
```

Comment:
Ideally this should have a comment `# runner config` to stay consistent with the above. Same elsewhere.